authors/apps/article/views.py

```diff
@@ -82,11 +95,11 @@ def perform_update(self, serializer):
         serializer.save(user=self.request.user)
 
 
-class ArticleRate(CreateAPIView):
+class ArticleRate(APIView):
```

Comment:
Why did you change this to APIView?

Reply:
@Eyansky Because is a get method not post